packages/api/src/derives/staking/index.ts

```diff
@@ -1 +1,11 @@
-export {queryStakingAccountDetails as accountInfo, querySession as sessionDetails} from './stakingAccount';
+export {
+  queryStakingAccountDetails as accountInfo,
+  querySession as sessionDetails,
+  DerivedStakingInfo,
+  DerivedSessionKeyInfo,
+} from './stakingAccount';
+export * from '@polkadot/api-derive/staking/account';
+export * from '@polkadot/api-derive/staking/controllers';
+export * from '@polkadot/api-derive/staking/electedInfo';
+export * from '@polkadot/api-derive/staking/overview';
+export * from '@polkadot/api-derive/staking/validators';
```

Comment:
Could we have a general practice guideline on how we are dealing with exporting `polkadot/api` in `cennznet/api`, like whether we encoverage consumers considering `cennznet/api` as the only api interface entry point, or suggest them use polkadot interfaces if possible where we only include cennznet customized api? @holygits

Reply:
@cennznet/api, users won't import @polkadot/api directly

Reply:
Ok, then lgtm